### PR TITLE
Popover fix

### PR DESCRIPTION
**Pull request by lizlizdesigns**
screen shot 2017-10-25 at 1 35 12 pm

### CODE REVIEW

**Comment by lizlizdesigns** [comment]
@lokesh - Ready for review please.

File: css/theme-weebly.scss [hunk 1]
```diff
@@ -931,4 +931,72 @@ div.Notice {
 	margin-right: 4px;
 }
 
+// Popover
```
Comment (lokesh) [inline css/theme-weebly.scss]: Can you move this CSS out of the Orbit Overrides section. I'd create a new Popover section below Cards.
`/* POPOVER ------------------------------------------------------------------- */`

File: css/theme-weebly.scss [hunk 2]
```diff
@@ -931,4 +931,72 @@ div.Notice {
...
+    position: absolute;
+    display: block;
+    width: 0;
+    height: 0;
+    border-color: #0000;
+    border-style: solid;
+}
+
+.popover-content {
+    padding: 9px 14px;
```
Comment (lokesh) [inline css/theme-weebly.scss]: Should we stick with our 4x multiples?
`padding: 12px 16px;`

**Review by lokesh — APPROVED** [review]
Two small comments. Looks good otherwise. Go ahead and merge after reviewing comments. Thanks!

File: css/theme-weebly.scss (hunk 2 above)
Comment (lokesh) [reply css/theme-weebly.scss]: It looks fine visually as-is. So your call.

**Pull request merged**